docs/available-filters/envelope-encryption.adoc

```diff
@@ -0,0 +1,155 @@
+:kms-api-javadoc: https://javadoc.io/doc/io.kroxylicious/kroxylicious-kms/latest
+
+= Envelope Encryption
+
+== What is it?
+
+A filter that provides a transparent encryption-at-rest solution for Apache Kafka.
```

Comment:
Maybe link to the use-case docs so the reader can understand more about the business problem being solved.